.annotation_safe_list.yml

```diff
@@ -21,20 +21,3 @@ student.Profile:
   ".. no_pii:": "This model has no PII (it's fake)"
 student.ProgramCourseEnrollment:
   ".. no_pii:": "This model has no PII (it's fake)"
-djcelery.CrontabSchedule:
-  ".. no_pii:": "This model has no PII"
-djcelery.IntervalSchedule:
-  ".. no_pii:": "This model has no PII"
-djcelery.PeriodicTask:
-  ".. no_pii:": "This model has no PII"
-djcelery.PeriodicTasks:
-  ".. no_pii:": "This model has no PII"
-djcelery.TaskMeta:
-  ".. no_pii:": "This model has no PII"
-djcelery.TaskSetMeta:
-  ".. no_pii:": "This model has no PII"
-djcelery.TaskState:
-  ".. no_pii:": "This model has no PII"
-djcelery.WorkerState:
-  ".. no_pii:": "This model has no PII"
```

Comment:
If we get rid of these models in edx-platform also, we should make a ticket for removing the tables afterwards (this would also merit a mention in the Koa release notes).